ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -183,11 +191,53 @@ - (BOOL)checkExtraQueryParameters
     return url!=nil;
 }
 
+- (BOOL)checkClaims:(ADAuthenticationError *__autoreleasing *)error
+{
+    if ([NSString adIsStringNilOrBlank:_claims])
+    {
+        return YES;
+    }
+    
```

Comment:
@oldalton Yep, current design is like that :), see point 2&3 below:

`1. We are not going fail the API if developers send empty or null claim in the new overload.`
`2. We are not going to do anything if developers send claims via extra qp in other overloads. `
`3. We are going to fail the API if developers send claims in both claims parameter and extra qp with the new overload. `